lib/daru/io/exporters/csv.rb

```diff
@@ -5,16 +5,28 @@ module IO
     module Exporters
       module CSV
```

Comment:
Seeing the below pseudocode also reminded me of @jeydurai's feature request (lazy call) at https://github.com/SciRuby/daru/issues/344 
```ruby
exporter = Daru::IO::Exporter::CSV.new(args) # parses and validates arguments
exporter.call # actually exports
```

Sure, let's go ahead with this - but I'm not sure if refactoring all of these changes is possible by Sunday (following the timeline). Nevertheless, I'll try my best. 😄